packages/application-shell/src/apollo-links/error-link.ts

```diff
@@ -36,8 +36,12 @@ const errorLink = onError(
       const context = operation.getContext() as TApolloContext;
 
       for (const err of graphQLErrors) {
+        const isNonAuthenticatedViaExtensionCode =
+          err?.extensions?.code === 'UNAUTHENTICATED';
+        const isNonAuthenticatedViaCode = err?.message === 'invalid_token';
```

Comment:
Should we mention here that this is a "workaround"? At least for us for future references or something.

Reply:
Added a comment.